controllers/apply/get-advice/form.js

```diff
@@ -0,0 +1,398 @@
+                cy: ''
+            }),
+            explanation: localise({
+                en: oneLine`If your organisation uses a different name in your
+                    day-to-day work, please write it below`,
+                cy: ``
+            }),
+            isRequired: false
+        }),
+        organisationAddress: {
+            name: 'organisationAddress',
+            schema: Joi.ukAddress().required(),
+            messages: [
+                {
+                    type: 'base',
+                    message: localise({
+                        en: 'Enter a full UK address',
+                        cy: 'Rhowch gyfeiriad Prydeinig llawn'
+                    })
+                },
+                {
+                    type: 'any.empty',
+                    key: 'line1',
+                    message: localise({
+                        en: 'Enter a building and street',
+                        cy: 'Rhowch adeilad a stryd'
+                    })
+                },
+                {
+                    type: 'any.empty',
+                    key: 'townCity',
+                    message: localise({
+                        en: 'Enter a town or city',
+                        cy: 'Rhowch dref neu ddinas'
+                    })
+                },
+                {
+                    type: 'any.empty',
+                    key: 'postcode',
+                    message: localise({
+                        en: 'Enter a postcode',
+                        cy: 'Rhowch gôd post'
+                    })
+                },
+                {
+                    type: 'string.postcode',
+                    key: 'postcode',
+                    message: localise({
+                        en: 'Enter a real postcode',
+                        cy: 'Rhowch gôd post go iawn'
+                    })
+                }
+            ]
+        },
+        organisationType: {
+            name: 'organisationType',
+            schema: Joi.string().required(),
+            messages: []
+        },
+        organisationBackground: {
+            name: 'organisationBackground',
+            schema: Joi.string()
+                .minWords(50)
+                .maxWords(500)
+                .required(),
+            messages: [
+                {
+                    type: 'base',
+                    message: localise({
+                        en: `Tell us about your organisation`,
+                        cy: ``
+                    })
+                },
+                {
+                    type: 'string.minWords',
+                    message: localise({
+                        en: `Answer must be at least 50 words`,
+                        cy: `Rhaid i’r ateb fod yn o leiaf 50 gair`
+                    })
+                },
+                {
+                    type: 'string.maxWords',
+                    message: localise({
+                        en: `Answer must be no more than 500 words`,
```

Comment:
Would be nice to see these as `settings` variables to make it easier to share the schema and the messages.

Reply:
Started doing that in a follow on PR so 👍